etl/PAIMS_PIMS/PAIMS_PIMS/MOTI_LEGACY_PAIMS_PIMS_DATA_MIGRATION.dtproj

```diff
@@ -41,12 +41,18 @@
                 <SSIS:Property SSIS:Name="Name">Package2</SSIS:Property>
                 <SSIS:Property SSIS:Name="VersionMajor">1</SSIS:Property>
                 <SSIS:Property SSIS:Name="VersionMinor">0</SSIS:Property>
+<<<<<<< HEAD
```

Comment:
@PryancaJSharma these are unresolved conflicts, and should never be merged. You'll either want to resolve these and then update this PR or remove these files from the PR.